source/includes/environment_config/_00-describe.md.erb

```diff
@@ -0,0 +1,8 @@
+<%=
+describe_object 'The environment config object', since: '16.6.0' do
+  name                      'String',  'The environment config name'
+  pipelines                 'Array',   'The list of pipeline names that belongs to the specified environment'
+  agents                    'Array',   'The list of agent uuids that belongs to the specified environment'
+  environment_variables     'Array',   'The list of [environment variables](#environment-config-environment-variables) that will be passed to all tasks (commands) that are part of this environment.'
```

Comment:
Is there any difference between ```pipeline-config-environment-variables``` and ```environment-config-environment-variables``` ? You can probably have a link to the ```pipeline-config-environment-variables``` and delete ```_01-environment-variable.md.erb``` file